docs/storefront/graphql/index.mdx

```diff
@@ -591,7 +591,7 @@ Use normal GraphQL Storefront API tokens. According to the [Principle of least p
 
 ### I want to run requests from a server, and I need to support customer login
 
-Use a Customer Impersonation token and store it securely on your server like other secrets. When you need to run requests in the context of a particular customer (for example, if they've logged in to your application), send their BigCommerce Customer ID along with the request as the `x-bc-customer-id` header.
+The recommended option is to use a regular storefront token with a customer access token. You will need to assign the customer access token to `X-Bc-Customer-Access-Token` in the header. The other possible but least preferred option is to use a customer impersonation token and store it securely on your server like other secrets. When you need to run requests in the context of a particular customer (for example, if they've logged in to your application), send their BigCommerce Customer ID along with the request as the `X-Bc-customer-id` header. This option provides less security but is available to use.
```

Comment:
🍹 
You will need to assign the customer access token to `X-Bc-Customer-Access-Token` in the header.

-->

You can exchange customer credentials for a customer access token using the login() mutation and send the token on future requests using the `X-Bc-Customer-Access-Token` header.